develop/plone/forms/wysiwyg.rst

```diff
@@ -120,11 +120,87 @@ You can use TinyMCE body selector make your CSS class have different styles in v
 
         Firefox does not actually support column breaks, so this was useful headaching experience.
 
+
 Customizing TinyMCE options
 ----------------------------
 
+Plone 4 uses TinyMCE 3. Plone 5 upgraded to TinyMCE 4, which works with a new concept called formats and therefore a new syntax for inline styles: `Your Custom Format's Title|custom_format_id|custom_icon_id`.
```

Comment:
I think so, because it's not really diving into Plone 4 explanations, it's just the subject introduction with a tiny bit of context/comparison on how things evolved. Probably useful for keyword searches too (e.g. by mentioning `tinymce.xml`).